package.json

```diff
@@ -3572,7 +3573,7 @@
         "buildScripts": "npm run generateClients && npm run generatePackage && npm run generateNonCodeFiles && npm run copyFiles",
         "buildBrowser": "npm run clean && npm run buildScripts && webpack --config webpack.browser.config.js --mode production && npm run copyFiles",
         "compile": "npm run clean && npm run buildScripts && webpack --mode development && npm run copyFiles",
-        "watch": "npm run clean && npm run buildScripts && tsc -watch -p ./",
+        "watch": "npm run clean && npm run buildScripts && webpack --config webpack.webview.config.js --mode development && tsc -watch -p ./",
```

Comment:
It's not needed for other webpack configs because those files are being emitted from the `tsc` command. And we can't rely on `tsc` for `webview.ts` because we want webpack to include its dependencies in the output

Reply:
How are other developers expected to understand these details? The differences are not obvious, and this configuration is fragile (easily broken in the future).

Why can't all the webviews work the same?

Reply:
Good point, let me see if there's another way to do this.